go/pkg/db/sqlc.json

```diff
@@ -23,7 +23,16 @@
           "output_db_file_name": "delete_me",
           "rename": {
             "Quotum": "Quotas"
-          }
+          },
+          "overrides": [
+            {
+              "db_type": "json",
+              "go_type": {
+                "type": "[]byte"
+              },
+              "nullable": true
+            }
```

Comment:
This override is added because sqlc doesn't support nullable json columns using json.RawMessage

It leads to those errors: `sql: Scan error on column index 3, name "meta": unsupported Scan, storing driver.Value type <nil> into type *json.RawMessage` 

See https://github.com/sqlc-dev/sqlc/issues/129